src/lib/evas/common/evas_image_scalecache.c

```diff
@@ -537,15 +533,11 @@ evas_common_rgba_image_scalecache_prepare(Image_Entry *ie, RGBA_Image *dst EINA_
    ret = SLKT(im->cache.lock);
    if (ret == EINA_FALSE) /* can't get image lock */
      {
-        useconds_t slp = 1, slpt = 0;
```

Comment:
I think instead we could create a `typedef` for `useconds_t` in Evil, what do you think? This way we ensure we're still using what posix expected :D

Reply:
Yeah, I agree